### PR TITLE
Small dev adjustments.

### PR DESCRIPTION
**Pull request by martinheidegger**
This PR does a few adjustments to the dev process.

- It replaces `npm-run-parallel` with `npm-run-all`. Because `npm-run-parallel` has bugs. − ceb2fe8
- It uses `optimization.nodeEnv` instead of the DefinePlugin to reduce the code. − ca58bb1

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ module.exports = (_, argv) => ({
   },
   plugins: [
     new DefinePlugin({
-      'process.env.NODE_ENV': argv.mode
+      'process.env.NODE_ENV': `"${argv.mode}"`
```
Comment (soyuka) [inline webpack.config.js]: why the change?

**Comment by martinheidegger** [comment]
@dkastl @soyuka I spent this afternoon trying to figure out why the node-env isn't replaced. In the end it turned out that that zombie-process was rewriting the bundle. I still changed some logic here cause it seems to be more effective.

**Comment by martinheidegger** [comment]
@soyuka Thanks for the approvement.
